packages/components/app/styles/components/dropdown.scss

```diff
@@ -25,311 +25,385 @@ $hds-dropdown-toggle-base-height: 36px;
 $hds-dropdown-toggle-border-radius: 5px;
 $hds-dropdown-toggle-border-width: 1px;
 
-.hds-dropdown {
 
-  // TOGGLE/ICON
-  .hds-dropdown-toggle-icon {
-    align-items: center;
+
+// TOGGLE/ICON
+.hds-dropdown-toggle-icon {
+  align-items: center;
+  background-color: transparent;
+  border: $hds-dropdown-toggle-border-width solid transparent; // We need this to be transparent for a11y
+  border-radius: $hds-dropdown-toggle-border-radius;
+  box-sizing: border-box; // TODO https://github.com/hashicorp/design-system-components/issues/46
+  color: var(--token-color-foreground-strong);
+  display: flex;
+  height: $hds-dropdown-toggle-base-height;
+  isolation: isolate;
+  justify-content: center;
+  outline-style: solid; // used to avoid double outline+focus-ring in Safari (see https://github.com/hashicorp/design-system-components/issues/161#issuecomment-1031548656)
+  outline-color: transparent; // We need this to be transparent for a11y
+  padding: 1px;
+  position: relative;
+  min-width: $hds-dropdown-toggle-base-height;
+
+  &:hover,
+  &.is-hover {
+    background-color: var(--token-color-surface-interactive);
+    border-color: var(--token-color-border-strong);
+    cursor: pointer;
+  }
+
+  &:focus,
+  &.is-focus {
+    @include hds-focus-ring-with-pseudo-element(
+      $top: -1px,
+      $right: -1px,
+      $bottom: -1px,
+      $left: -1px,
+      $radius: $hds-dropdown-toggle-border-radius
+    );
+  }
+
+  &:active,
+  &.is-active {
+    background-color: var(--token-color-surface-interactive-active);
+    border-color: var(--token-color-border-strong);
+  }
+}
+
+.hds-dropdown-toggle-icon__wrapper {
+  align-items: center;
+  border-radius: 3px; // $hds-dropdown-toggle-border-radius- 1px padding - 1px border
+  display: flex;
+  justify-content: center;
+  height: 32px;
+  padding: 1px;
+  width: 32px;
+
+  img {
+    border-radius: inherit;
+  }
+}
+
+.hds-button__icon { // TODO this class is not being used.
+  margin-right: -6px; // we apply a negative margin to counter the padding-right of the button and reduce the visual space between the icon and the right border
+  margin-left: 0.5rem; // this overrides the rule `.hds-button__text + .hds-button__icon`
+}
```

Comment:
this can be removed, it was in the `.hds-dropdown-toggle--with-button-component` (see lines `100-103`)